CONTRIBUTING.md

```diff
@@ -3,125 +3,96 @@
-4. Ensure the test suite passes (`yarn build && yarn test`).
+1. Fork the repository and create your branch.
+2. Follow the setup instructions in this file.
+3. If you’re fixing a bug or adding code that should be tested, add tests!
+4. Ensure the test suite passes.
 
 ## Developing locally
 
-During the process of developing the library locally we first have to check out the repository and create a branch from main.
+First, clone the `hotwired/turbo` repository and install dependencies:
 
 ```bash
 git clone https://github.com/hotwired/turbo.git
-cd turbo
-yarn
 ```
 
 ```bash
-git checkout -b '<your_branch_name>'
+cd turbo
+yarn install
 ```
 
-Once you are done developing the feature or bug fix you have 2 options:
+Then create a branch for your changes:
 
-1. Run the test suite
-2. Run a local webserver and checkout your changes manually
+```bash
+git checkout -b <your_branch_name>
+```
 
 ### Testing
 
-The library is tested by running the test suite (found in: `src/tests/*`) against headless browsers. The browsers are setup in [web-test-runner.config.mjs](./web-test-runner.config.mjs) and [playwright.config.ts](./playwright.config.ts). Check them out to see the used browser environments.
+Tests are run through `yarn` using [Web Test Runner](https://modern-web.dev/docs/test-runner/overview/) with [Playwright](https://github.com/microsoft/playwright) for browser testing. Browser and runtime configuration can be found in [`web-test-runner.config.mjs`](./web-test-runner.config.mjs) and [`playwright.config.ts`](./playwright.config.ts).
 
-To override the ChromeDriver version, declare the `CHROMEVER` environment
-variable.
-
-First, install the drivers to test the suite in browsers:
+To begin testing, install the browser drivers:
 
 ```bash
-yarn playwright install  --with-deps
+yarn playwright install --with-deps
 ```
 
-The tests are using the compiled version of the library and they are themselves also compiled. To compile the tests and library and watch for changes:
+Then build the source. Because tests are run against the compiled source (and are themselves compiled) be sure to run `yarn build` prior to testing. Alternatively, you can run `yarn watch` to build and watch for changes.
 
 ```bash
-yarn watch
+yarn build
 ```
 
-To run the unit tests:
+### Running the test suite
+
+The test suite can be run with `yarn`, using the test commands defined in [`package.json`](./package.json). To run all tests in all configured browsers:
 
 ```bash
-yarn test:unit
+yarn test
 ```
 
-To run the browser tests:
+To run just the unit or browser tests:
 
 ```bash
+yarn test:unit
 yarn test:browser
 ```
 
-To run the browser suite against a particular browser (one of
-`chrome|firefox`), pass the value as the `--project=$BROWSER` flag:
+By default, tests are run in "headless" mode against all configured browsers (currently `chrome` and `firefox`). Use the `--headed` flag to run in normal mode. Use the `--project` flag to run against a particular browser.
```

Comment:
> currently `chrome` and `firefox`

We're running _unit_ tests against `webkit` but not others (see "projects" in `playwright.config.ts`). Everything passes for me on webkit, so I was curious if there's a reason for the omission. Does anyone know?